pkg/node/node_address.go

```diff
@@ -437,6 +437,11 @@ func getCiliumHostIPs() (ipv4GW, ipv6Router net.IP) {
 	nodeConfig := option.Config.GetNodeConfigPath()
 	ipv4GW, ipv6Router = getCiliumHostIPsFromFile(nodeConfig)
 	if ipv4GW != nil {
+		log.WithFields(logrus.Fields{
+			"ipv4": ipv4GW,
+			"ipv6": ipv6Router,
```

Comment:
Can `ipv4GW` be nil but `ipv6Router` non-nil here?

Reply:
Good point. Fixed